crates/pera-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -88,10 +96,25 @@ impl DWalletMPCManager {
             weighted_parties.clone(),
         )
         .map_err(|_| PeraError::InternalDWalletMPCError)?;
+
+        // Start the network DKG if this is the first epoch
+        let (status, mpc_instances) = if epoch_id == FIRST_EPOCH_ID {
+            (
+                ManagerStatus::WaitingForNetworkDKGCompletion,
+                NetworkDkg::init(epoch_store.clone())?,
+            )
+        } else {
+            (ManagerStatus::Active, HashMap::new())
+        };
+        let mut outputs_manager = epoch_store.get_dwallet_mpc_outputs_manager().await?;
```

Comment:
Also if this is not the first epoch we need to fetch the validator secret from the chain here, so maybe add an additional status & a TODO here for that